src/core/execute.c

```diff
@@ -396,6 +396,7 @@ int exec_spawn(
         assert(ret);
 
         LOG_CONTEXT_PUSH_UNIT(unit);
+        LOG_CONTEXT_SET_LOG_LEVEL(context->log_level_max >= 0 ? context->log_level_max : log_get_max_level());
```

Comment:
doesn't this already happen by default? the macro do look at log_level_max from the unit?

Reply:
`log_unit_debug()` might look at it, but `log_debug()` doesn't, and guess which one `open_serialization_fd()` uses?

Reply:
The most useful one I am sure

Reply:
If those fail we should and already complain loud? I think whether the util lib funcs open_serialization_fd calls should be attributed to the unit itself is debatable...

Reply:
Actually yes everything to do with setting up and spawning a unit should be marked as such, so that with my other PR I can get full logs out of that too on restart failures. The non-debug log messages are very often very inadequate to actually debug things.

Reply:
@YHNdnzj That's already happening regardless of this change though. The only thing this changed is that the debug messages are attributed to the unit as well.